packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -408,6 +383,12 @@
       <artifactId>hive-service</artifactId>
       <version>${hive.version}</version>
       <scope>${utilities.bundle.hive.scope}</scope>
+      <exclusions>
```

Comment:
hive-* and hbase-* dependencies are already shaded. wouldn't that be sufficient? why do we need to exclude here?

Reply:
Hive brings its own HBase deps, that collied w/ the ones we depend on (we need 2.4.9, Hive brings 1.1.1)